pretender.js

```diff
@@ -489,4 +495,4 @@ if (typeof module === 'object') {
   });
 }
 self.Pretender = Pretender;
-}(self));
+}(typeof self !== 'undefined' ? self : this));
```

Comment:
We should use something more robust (and likely call it `global`).

For example, I use something like this in a few projects:

```js
(function(global) {

}(() => {
  if (typeof self !== 'undefined') {
    return self;
  } else if (typeof window !== 'undefined') {
    return window;
  } else if (typeof global !== 'undefined') {
    return global;
  } else {
    return Function('return this')();
  }
})());
```